spec/dummy/app/models/dummy_user.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 class DummyUser < ActiveRecord::Base
+  def email
+    'user@example.org'
+  end
```

Comment:
The DemoMode CLI expects the `signinable` to have an email, but the `DummyUser` in our test personas doesn't actually have an email attribute.